src/main/scala/circt/stage/phases/CIRCT.scala

```diff
@@ -148,8 +148,20 @@ class CIRCT extends Phase {
 
     try {
       logger.info(s"""Running CIRCT: '${cmd.mkString(" ")} < $$input'""")
-      val result = (cmd #< new java.io.ByteArrayInputStream(input.getBytes)).!!
-
+      val stdoutStream = new java.io.ByteArrayOutputStream
+      val stderrStream = new java.io.ByteArrayOutputStream
+      val stdoutWriter = new java.io.PrintWriter(stdoutStream)
+      val stderrWriter = new java.io.PrintWriter(stderrStream)
+      val exitValue = (cmd #< new java.io.ByteArrayInputStream(input.getBytes)).!(ProcessLogger(stdoutWriter.println, stderrWriter.println))
+      stdoutWriter.close()
+      stderrWriter.close()
+      val result = stdoutStream.toString
+      val errors = stderrStream.toString
+      if (exitValue != 0) {
+        //logger.error(errors)
+        StageUtils.dramaticError(s"${binary} failed.\nExitCode:\n${exitValue}\nSTDOUT:\n${result}\nSTDERR:\n${errors}")
```

Comment:
It might be cleaner to change this to throw a `java.io.IOException`. In the catch below, the current `java.io.IOException` should probably be a `java.io.FileNotFoundException`, but I'd have to see what actually gets thrown. This would then prevent doing error handling code in the `try` portion. This would require moving the streams outside the `try` so they are available to both.

`logger.error` is probably better, but the current code doesn't use this. I propose using `StageError` for now and changing it to `logger.error` in a subsequent patch.